### DISCUSSION TITLE
Hi, any ideas on things you want to improve?

### DISCUSSION THREAD

**Discussion by JayLog22** [Leo-Corporation/Passliss#149] [Ideas]
Hi there! I'd like to contribute but not sure what you're aiming for or if you have something in mind you'd like to improve.

Any ideas?

**Comment by lpeyr** [comment]
Hello, welcome to the project!
Sorry for the late reply, I only saw your question now. I don't have anything specific planned for the future: When I roll out updates, I just add whatever ideas come to mind. If you have any, please let me know! There are several things you can contribute to, like localized resources/translations (for a better experience, you need to have the ["ResXManager"](https://marketplace.visualstudio.com/items?itemName=TomEnglert.ResXManager) extension installed to be able to edit the translations.), or the system that rates password strength, which is available here:

https://github.com/Leo-Corporation/Passliss/blob/4f82c06df97bd0a0657d210dc942cf5c2c0691f9/Passliss/Classes/Global.cs#L242-L324

I wrote it a while ago, it works, but it could be improved.
